src/store/eventing/types.ts

```diff
@@ -4,7 +4,12 @@ import { Annotation, NewAnnotation } from '../../@types';
 
 export enum Event {
     ANNOTATOR = 'annotatorevent',
-    CREATE_ANNOTATION = 'annotationCreate',
+    CREATE_ANNOTATION = 'annotationcreate',
+    ERROR = 'annotationerror',
+    SCALE = 'scaleannotations',
```

Comment:
Can we add a comment to the effect of this being a "legacy" event and that the string literal should not be changed?